### PR TITLE
Update doc

### PR DESCRIPTION
**Pull request by hiroyuki-sato**
What do you think this translation?

```
+msgid "* Stopped to register auto test runner."
+msgstr "* AutoRunner自動登録の停止"
```

Additionally I fixed typo Improvments -> Improvements.

### CODE REVIEW

File: doc/po/ja.po [hunk 1]
```diff
@@ -187,23 +167,22 @@ msgstr ""
...
-#: ../../doc/text/news.textile:18
 msgid "h2(#1-0-3). 1.0.3 - 2012-11-29"
 msgstr ""
 
-#: ../../doc/text/news.textile:20
 msgid "A support Capybara 2.0.1 release."
 msgstr "Capybara 2.0.1 サポートのリリースです。"
 
-#: ../../doc/text/news.textile:24
+#, fuzzy
```
Comment (kou) [inline doc/po/ja.po]: We need to remove this line.

File: doc/po/ja.po [hunk 2]
```diff
@@ -136,46 +121,41 @@ msgid ""
...
 
-#: ../../doc/text/news.textile:3
+msgid "h2(#1-0-5). 1.0.5 - 2016-01-18"
+msgstr ""
+
+msgid "h3. Improvements"
+msgstr "h3. 改良"
+
+msgid "* Stopped to register auto test runner."
+msgstr "* AutoRunner自動登録の停止"
```
Comment (kou) [inline doc/po/ja.po]: How about "* 自動テスト実行機能の登録をやめた"?

Comment (hiroyuki-sato) [reply doc/po/ja.po]: Thank you!. Done.

File: doc/po/ja.po (hunk 1 above)
Comment (hiroyuki-sato) [reply doc/po/ja.po]: Thank you. Removed.

**Comment by hiroyuki-sato** [comment]
@kou PTAL 
BTW When I execute `rake reference:pot:generate` command again, 
It generate the following lines again.
Can I fix it?

```
+#, fuzzy
+msgid "h3. Improvments"
+msgstr "h3. 改良"
+
```

**Comment by kou** [comment]
You should change `edit.po` not `po`.

**Comment by hiroyuki-sato** [comment]
Thank you!

**Pull request merged**